### DISCUSSION TITLE
How can I solve Error establishing a database connection

### DISCUSSION THREAD

**Discussion by iremetech** [EasyEngine/easyengine#1786] [Support Requests]
Hello, last year I migrated my wordpress websites from shared hosting to digitalocean nginx server using EasyEngine to install WP sites, everything was working well until yesterday but today all my websites on my server are currently showing 'Error establishing a database connection'. I successfully restart the DB by running the 'ee service restart db' command but I still get the same error, and when I run ee service enable db, I get 'Service global-db already enabled', when I run df -h command, down is what I get. 

![DO](https://user-images.githubusercontent.com/127326366/233804477-99485325-b6bf-4074-8b8b-ea682fa845e7.PNG)

What could be the optimal solution to solve this?

**Comment by red0888** [comment]
Delete those overlay files with ftp and after that restart containers
